rest-api/src/main/scala/za/co/absa/enceladus/rest_api/services/DatasetService.scala

```diff
@@ -93,7 +94,15 @@ class DatasetService @Autowired()(datasetMongoRepository: DatasetMongoRepository
   }
 
   override def getUsedIn(name: String, version: Option[Int]): Future[UsedIn] = {
-    Future.successful(UsedIn())
+    val existingEntityF = version match {
```

Comment:
I prefer without the type initial in the name

```suggestion
    val existingEntity = version match {
```

Reply:
Sure, that's better.